Graphics/GraphicsEngineOpenGL/CMakeLists.txt

```diff
@@ -219,6 +231,11 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
             COMPILE_FLAGS " -Wno-unused-variable"
         )
     endif()
+    if(PLATFORM_EMSCRIPTEN)
+        set_property(TARGET Diligent-GraphicsEngineOpenGL-static APPEND_STRING PROPERTY
+            LINK_FLAGS "-s FULL_ES3=1 -s"
```

Comment:
Does it works?
Docs says:
"Additional flags to use when linking this target if it is a shared library, module library, or an executable. Static libraries need to use STATIC_LIBRARY_OPTIONS or STATIC_LIBRARY_FLAGS properties."